devassistant/command_runners.py

```diff
@@ -456,23 +469,25 @@ def _dot_devassistant_write(cls, comm):
 
 @register_command_runner
 class GitHubCommandRunner(CommandRunner):
-    _user = None
-    try:
-        _gh_module = utils.import_module('github')
-    except:
-        _gh_module = None
     _required_yaml_args = {'default': ['login', 'reponame'],
                            'create_repo': ['login', 'reponame', 'private'],
                            'create_and_push': ['login', 'reponame', 'private'],
                            'create_fork': ['login', 'repo_url'],
                            'push': []}
 
+    def __init__(self, c):
+        self.c = c
+        self._user = None
+        try:
+            self._gh_module = utils.import_module('github')
```

Comment:
Is it possible, that some instances will import the module and some will fail? IS there any logical reason to keep this per instance?